cli/tests/pcluster/aws/test_ec2.py

```diff
@@ -210,3 +210,29 @@ def test_get_ebs_snapshot_info(boto3_stubber, snapshot_id, error_message):
         with pytest.raises(AWSClientError, match=error_message) as clienterror:
             Ec2Client().get_ebs_snapshot_info(snapshot_id)
             assert_that(clienterror.value.code).is_not_equal_to(0)
+
+
+@pytest.mark.parametrize(
+    "error_code, raise_exception",
+    [("DryRunOperation", False), ("UnsupportedOperation", True)],
+)
+def test_run_instances_dryrun(boto3_stubber, error_code, raise_exception):
+    """Verify that if run_instance doesn't generation exception if the error code is DryRunOperation."""
```

Comment:
Typo in the comment. Should be "Verify that run_instance doesn't generate exception if the error code is DryRunOperation."

Reply:
Done